src/Entity/DoofinderAdminPanelView.php

```diff
@@ -569,4 +575,39 @@ private function getBooleanFormValue()
 
         return $option;
     }
+
+    private function getFeedURLs()
+    {
+        $urls = [];
+        $context = \Context::getContext();
+        $languages = \Language::getLanguages(true, $context->shop->id);
+        foreach ($languages as $lang) {
+            foreach (\Currency::getCurrencies() as $cur) {
+                $currencyIso = \Tools::strtoupper($cur['iso_code']);
+                $langIso = \Tools::strtoupper($lang['iso_code']);
+                $urls[] = [
+                    'url' => UrlManager::getFeedUrl($context->shop->id, $langIso, $currencyIso),
+                    'lang' => $langIso,
+                    'currency' => $currencyIso,
+                ];
+            }
+        }
+
+        return $urls;
+    }
+
+    private function feedUrlsFormatHtml($df_feed_urls)
+    {
+        $htmlContent = '<dl>';
+        foreach ($df_feed_urls as $feed_url) {
+            $htmlContent .= '<dt>' . $this->module->l('Data feed URL for', 'doofinder') . ' ['
```

Comment:
```suggestion
            $htmlContent .= '<dt>' . $this->module->l('Data feed URL for', 'doofinderadminpanelview') . ' ['
```

This will require a regeneration of the translations. Otherwise, the string won't be translated